charmhelpers/core/host.py

```diff
@@ -114,6 +114,17 @@ def service_stop(service_name, **kwargs):
     return service('stop', service_name, **kwargs)
 
 
+def service_enable(service_name):
+    """Enable a system service.
+
+    The specified service name is managed via the system level init system.
+    Enabling a service allows it to run automatically on boot.
+
+    :param service_name: the name of the service to enable
+    """
+    return service('enable', service_name)
```

Comment:
So, I've made a mistake and need to ask you to change direction on this please.  I hadn't looked sufficiently around the code to notice that I was steering you in the wrong direction, for which I apologise, particularly as I'm asking you to re-work the patch please.

Essentially, looking at the other functions (`service_restart`, etc) the `**kwargs` argument is passed to the `service(...)` function and *may* have effects there.

So the ask is to restore the `**kwargs` to the function signature *and* pass them to as `service('enable', service_name, **kargs)`.

This also means restoring the docstring (a cut and paste from one of the other functions), as well.

Thanks very much!